Sources/AlgoliaSearch-Client/FilterBuilder/Proxies/OrGroupProxy.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  OrGroupProxy.swift
+//  AlgoliaSearch OSX
+//
+//  Created by Vladislav Fitc on 24/12/2018.
+//  Copyright © 2018 Algolia. All rights reserved.
+//
+
+import Foundation
+
+/// Provides a specific type-safe interface for FilterBuilder specialized for a disjunctive group
+
+public struct OrGroupProxy<T: Filter>: GroupProxy {
+    
+    let filterBuilder: FilterBuilder
+    let group: AnyFilterGroup
+    
+    public var isEmpty: Bool {
```

Comment:
Docs missing here

also, maybe it's better to make it as a function instead of a variable? WDYT?

Reply:
I prefer to keep it as a variable. 
1. It's done the same way in Swift standard library for Sequence/Collection 
2. There is an O(1) operation behind it, so no need to imply developer to avoid it